packages/tools/src/types/LabelmapTypes.ts

```diff
@@ -35,7 +35,18 @@ export type LabelmapRenderingConfig = {
   ofun?: vtkPiecewiseFunction;
 };
 
-export type LabelmapSegmentationData = {
+export type LabelmapSegmentationDataVolume = {
+  type: 'volume';
```

Comment:
I think we can check if there exists `volumeIds` or `imageIds` then infer the type instead of having explicit type here, no?

Reply:
But how that way there isnt no typescript checking. I am ok if you are

Reply:
you can check if volumeId is present or imageIds. At least make the type: 'stack'